app/controllers/register/leadtrustee/organisation/nonTaxable/CountryOfResidenceController.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.register.leadtrustee.organisation.nonTaxable
+
+import config.FrontendAppConfig
+import config.annotations.LeadTrusteeOrganisation
+import controllers.actions.StandardActionSets
+import controllers.actions.register.leadtrustee.organisation.NameRequiredActionImpl
+import forms.CountryFormProvider
+import navigation.Navigator
+import pages.register.leadtrustee.organisation.nonTaxable.CountryOfResidencePage
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.RegistrationsRepository
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import utils.countryOptions.CountryOptionsNonUK
+import views.html.register.leadtrustee.organisation.nonTaxable.CountryOfResidenceView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class CountryOfResidenceController @Inject()(
+                                              override val messagesApi: MessagesApi,
+                                              registrationsRepository: RegistrationsRepository,
+                                              @LeadTrusteeOrganisation navigator: Navigator,
+                                              standardActionSets: StandardActionSets,
+                                              nameAction: NameRequiredActionImpl,
+                                              formProvider: CountryFormProvider,
+                                              val controllerComponents: MessagesControllerComponents,
+                                              view: CountryOfResidenceView,
+                                              val countryOptions: CountryOptionsNonUK
+                                            )(implicit ec: ExecutionContext, appConfig: FrontendAppConfig) extends FrontendBaseController with I18nSupport {
+
+  private val form: Form[String] = formProvider.withPrefix("leadTrustee.organisation.nonTaxable.countryOfResidence")
```

Comment:
Likewise with the message keys, this is for 5mld, it isn't specific to non-taxable?